specification/netapp/resource-manager/Microsoft.NetApp/stable/2020-06-01/netapp.json

```diff
@@ -3394,9 +3394,6 @@
           "type": "string",
           "readOnly": true,
           "description": "Unique Baremetal Tenant Identifier.",
-          "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
-          "maxLength": 36,
-          "minLength": 36,
```

Comment:
Why are we changing this in the previous version?

Reply:
We thought to correct and align api versions as this should not have been in there.